src/DataCollector/index.js

```diff
@@ -30,14 +32,19 @@ const mapDispatchToProps = (dispatch) => ({
     dispatch(makePredictions(config));
   },
   resetPrediction: () => dispatch(resetPrediction()),
-  setGroundTruth: (gt) => dispatch(setGroundTruth(gt)),
-  addItemsToReciept: (gt, rawPrediction) => {
-    if (_.isEmpty(gt)) {
+  setGroundTruth: (gt, index) => dispatch(setGroundTruth(gt, index)),
+  addItemsToReciept: (gts, rawPrediction) => {
+    const label = gts
+      // eslint-disable-next-line camelcase
```

Comment:
I can't check it but I think it is redundant. This rule is for variables names and not object fields

Reply:
Yes, it looks redundant but there is an eslint error without it